IntegrationAdaptersTests/SftpTests.cs

```diff
@@ -9,12 +12,37 @@ namespace IntegrationAdaptersTests
 {
     public class SftpTests
     {
+        public Mock<IMedicineReportRepository> mock;
         [Fact]
         public void sendFileSuccess()
         {  
             SftpService service = new SftpService();
             bool success = service.SendFile(@"C:\Users\dragana\Desktop\Projekat PSW\Healthcare-System\IntegrationAdaptersTests\bin\Debug\netcoreapp3.1\testFile.txt");
             Assert.True(success);   
         }
+
+        [Fact]
+        public void Find_existing_report()
+        {
+            mock = new Mock<IMedicineReportRepository>();
+            mock.Setup(expression: t => t.GetAll()).Returns(new List<MedicineReport> { new MedicineReport { Id = "1", Date = new DateTime(2020, 10, 5), Dosage = new List<MedicineDosage>() }, new MedicineReport { Id = "2", Date = new DateTime(2020, 5, 10), Dosage = new List<MedicineDosage>() } });

```

Comment:
If you're using objects for stubs, move them to TestConstants class in util package or make fields in this class (if they're used only in this test)